---
layout: default
permalink: previous.html
---

## Previous Projects and Affiliations

### UC Berkeley - Coates Lab

I did my PhD research in the lab of [Dr. John Coates](http://coateslab.berkeley.edu/) at UC Berkeley in the [Plant and Microbial Biology Department](http://plantandmicrobiology.berkeley.edu/). In the Coates Lab, I worked on the genetics and genomics of bacterial perchlorate metabolism.

During my time in the Coates Lab, I identified a horizontally transferred [genomic island containing the genes for perchlorate reduction](http://aem.asm.org/content/77/20/7401.full) as well as several genes with unknown function. I also developed the [first genetic system for a perchlorate reducer](http://mbio.asm.org/content/5/1/e00769-13.abstract) using the model organism *Azospira suillum* PS and identified genes essential for perchlorate reduction. Using a mixture of biochemistry, next-gen sequencing methods like RNA-seq, and classical genetics I characterized [a novel mechanism for amelioration of periplasmic hypochlorite stress](http://mbio.asm.org/content/6/3/e00233-15.full). Throughout my time in the Coates Lab, I was involved in sequencing and analyzing the genomes of perchlorate-reducing isolates, which culminated in an improved understanding of the [diversity and evolution](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4623255/) of this metabolism.

### LBNL - Arkin Lab

From 2014 to 2015, I worked in the [lab of Dr. Adam Arkin](http://genomics.lbl.gov/) at LBNL. I worked on several projects there, including synthetic ecology and directed evolution of *Pseudomonas* isolates as well as the random barcoded transposon sequencing project (RB-TnSeq) led by Dr. Adam Deutschbauer. I built RB-TnSeq libraries for several model plant-associated bacteria, including *Sinorhizobium meliloti* 1021, *Burkholderia phytofirmans* PsJN, and *Herbaspirillum seropedicae* SmR1. Using these libraries along with those from several dozen other bacteria, thousands of independent fitness assays were carried out, generating [a massive public bacterial fitness database](http://fit.genomics.lbl.gov) and [annotations for thousands of previously unknown bacterial genes](https://www.nature.com/articles/s41586-018-0124-0).

### Bayer Crop Science

From 2015 to 2016, I worked in the [Biologicals R&D division of Bayer Crop Science](https://www.research.bayer.com/en/biological-crop-protection.aspx) at the West Sacramento campus. I was in the Computational Life Sciences group, where I worked on various projects involving genomics-driven trait discovery for agricultural applications of bacteria.


[back](./)
